### PR TITLE
docs: update client examples & add sid explanation

### PR DESCRIPTION
**Pull request by ilgooz**

### CODE REVIEW

**Comment by antho1404** [comment]
Do we need to keep the go version ? I would just have the js for now.
@NicolasMahe what do think about that ?

**Comment by NicolasMahe** [comment]
Let's keep the go version as well. We should have example with ALL gRPC generated stuff (event PHP....)

File: README.md [hunk 1]
```diff
@@ -97,10 +97,14 @@ const invitation = '__ID_SERVICE_INVITATION_DISCORD__' // To replace by the Serv
...
-  { serviceID: invitation, taskKey: 'send', inputs: { email, sendgridAPIKey } },
-)
+MESG.listenEvent({ serviceID: webhook, eventFilter: 'request' })
+  .on('data', (event) => {
+    MESG.executeTask({
+      serviceID: invitation,
+      taskKey: 'send',
+      inputData: JSON.stringify({ email, sendgridAPIKey })
+    })
+  })
```
Comment (NicolasMahe) [inline README.md]: What about adding error management that output the error?
The doc should show a basic error management to push dev to catch error.

Comment (ilgooz) [reply README.md]: done

File: docs/tutorials/erc20-transfer-notifications/receive-email-when-there-is-an-erc20-transfer.md [hunk 2]
```diff
@@ -124,7 +123,14 @@ const MESG = require('mesg-js').application()
 Then simply add the `whenEvent` function at the end of your code to link the event to the task:
 
 ```javascript
-MESG.whenEvent(erc20Transfer, sendEmail)
+MESG.listenEvent(erc20Transfer)
+  .on('data', (event) => {
+    const transfer = JSON.parse(event.eventData)
+    console.log('New ERC20 transfer received. will send an email. Transaction hash:', transfer.transactionHash)
+    MESG.executeTask(emailTask(transfer)).catch((err) => console.log(err.message))
```
Comment (NicolasMahe) [inline docs/tutorials/erc20-transfer-notifications/receive-email-when-there-is-an-erc20-transfer.md]: Let's put the code of `emailTask(transfer)` in.
It's was logic with the previous lib to extract event and task definition, but it's not anymore

Comment (ilgooz) [reply docs/tutorials/erc20-transfer-notifications/receive-email-when-there-is-an-erc20-transfer.md]: done

**Pull request merged**